scripts/SREP/get-sosreport/metadata.yaml

```diff
@@ -0,0 +1,37 @@
+file: script.sh
+name: get-sosreport
+description: Collect a sosreport from a specific node
+author: Hevellyn Gomes 
+allowedGroups:
+  - SREP
+  - LPSRE
+  - CEE
```

Comment:
Just FYI, since our backplane-api has supported to have a cluster version validation feature for managed-script and our `metadata.schema` has updated accordingly, you can add `clusterVersions` key right below the `allowedGroups` to make the script only run against a validated cluster version you defined in this metadata.yaml as needed. (It's an optional though)
```
allowedGroups:
  - SREP
  - LPSRE
  - CEE
clusterVersions:
  - 4.13.5